src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -63,11 +63,13 @@ const ScenarioSidebar: React.FC<Props> = (props) => {
           name={name}
           setName={setName}
           placeholderValue={scenario?.name}
```

Comment:
I needed to optionally pass a persistence method in because depending on the context we're in we may or may not actually want to save the inputs immediately.  For example, when changes are made in the Sidebar for the Scenario, we won't those changes saved.  When changes are made on the Facility Details page (where these components are also used), we don't want those changes persisted until the user clicks the Save button.

I've tested and verified that both of these use cases now function properly.